Name: dialingshortcut-assistant.js
Version: 2.1.0-1
Author: The Resetter
Description: Add 3, 8 and 10 Digits to Dialing Shortcuts

--- .orig/usr/palm/applications/com.palm.app.phoneprefs/app/controllers/dialingshortcut-assistant.js
+++ /usr/palm/applications/com.palm.app.phoneprefs/app/controllers/dialingshortcut-assistant.js
@@ -8,10 +8,12 @@
     setup: function(){
 		this.digitList = [{label:$L('3 digits'), value:"3DigitNumber"},
 		                  	{label:$L('4 digits'), value:"4DigitNumber"},
-							{label:$L('5 digits'), value:"5DigitNumber"},
-		 					{label:$L('6 digits'), value:"6DigitNumber"},
-							{label:$L('7 digits'), value:"7DigitNumber"}
-						 ];
+					{label:$L('5 digits'), value:"5DigitNumber"},
+ 					{label:$L('6 digits'), value:"6DigitNumber"},
+					{label:$L('7 digits'), value:"7DigitNumber"},
+					{label:$L('8 digits'), value:"8DigitNumber"},
+					{label:$L('10 digits'), value:"10DigitNumber"}
+				 ];
 		this.digitListModel = {
 				currentSelection: '4DigitNumber',
 				choices: this.digitList
